Library/Formula/rubberband.rb

```diff
@@ -0,0 +1,169 @@
+
+class Rubberband < Formula
+  homepage "http://breakfastquay.com/rubberband/"
+  url "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2"
+  version "1.8.1"
+  sha256 "ff0c63b0b5ce41f937a8a3bc560f27918c5fe0b90c6bc1cb70829b86ada82b75"
+
+  depends_on "libsndfile"
+
+  patch :p1, :DATA
+
+  def install
+    args = %W[
+      CC=#{ENV.cc}
+      CXX=#{ENV.cxx}
+      PREFIX=#{prefix}
+    ]
+    system "make", "--file=Makefile.osx", "install", *args
+  end
+end
+
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!